app/src/main/kotlin/com/simplemobiletools/clock/extensions/Activity.kt

```diff
@@ -0,0 +1,32 @@
+package com.simplemobiletools.clock.extensions
+
+import com.simplemobiletools.clock.BuildConfig
+import com.simplemobiletools.commons.activities.BaseSimpleActivity
+import com.simplemobiletools.commons.dialogs.PermissionRequiredDialog
+import com.simplemobiletools.commons.extensions.canUseFullScreenIntent
+import com.simplemobiletools.commons.extensions.openFullScreenIntentSettings
+
+fun BaseSimpleActivity.handleFullScreenNotificationsPermission(
+    notificationsCallback: (granted: Boolean, fullScreenGranted: Boolean) -> Unit,
```

Comment:
This is still quite confusing. Shouldnt we treat both the same? Like if either casual or fullscreen permission is declined, we disable the given feature, or whatever was the user doing?